### ISSUE TITLE
Post Install script fails on El Capitan beta

### ISSUE THREAD

**Issue by thommay** [chef/omnibus-chef#456]
```
Aug 17 17:00:14 remtmay02 installd[428]: PackageKit: Adding client PKInstallDaemonClient pid=858, uid=503 (/System/Library/CoreServices/Installer.app/Contents/MacOS/Installer)
Aug 17 17:00:14 remtmay02 Installer[858]: PackageKit: Enqueuing install with framework-specified quality of service (utility)
Aug 17 17:00:14 remtmay02 installd[428]: PackageKit: ----- Begin install -----
Aug 17 17:00:14 remtmay02 installd[428]: PackageKit: request=PKInstallRequest <1 packages, destination=/>
Aug 17 17:00:14 remtmay02 installd[428]: PackageKit: packages=(
	    "PKLeopardPackage <file://localhost/Volumes/Chef%20Development%20Kit/chefdk-0.7.0-1.pkg#chefdk-core.pkg>"
	)
Aug 17 17:00:14 remtmay02 installd[428]: PackageKit: Will do receipt-based obsoleting for package identifier com.getchef.pkg.chefdk (prefix path=opt/chefdk)
Aug 17 17:00:15 remtmay02 Installer[858]: LSExceptions [0x7fd641703540] unloaded
Aug 17 17:00:15 remtmay02 installd[428]: PackageKit: Extracting file://localhost/Volumes/Chef%20Development%20Kit/chefdk-0.7.0-1.pkg#chefdk-core.pkg (destination=/var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/C/PKInstallSandboxManager/9E06E7DA-B6EA-46C6-9D5F-6AB499302301.activeSandbox/Root/opt/chefdk, uid=0)
Aug 17 17:00:23 remtmay02 installd[428]: PackageKit: prevent user idle system sleep
Aug 17 17:00:23 remtmay02 installd[428]: PackageKit: suspending backupd
Aug 17 17:00:23 remtmay02 installd[428]: PackageKit: Using trashcan path /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/PKInstallSandboxTrash/9E06E7DA-B6EA-46C6-9D5F-6AB499302301.sandboxTrash for sandbox /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/C/PKInstallSandboxManager/9E06E7DA-B6EA-46C6-9D5F-6AB499302301.activeSandbox
Aug 17 17:00:23 remtmay02 install_monitor[863]: Temporarily excluding: /Applications, /Library, /System, /bin, /private, /sbin, /usr
Aug 17 17:00:23 remtmay02 installd[428]: PackageKit: Shoving /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/C/PKInstallSandboxManager/9E06E7DA-B6EA-46C6-9D5F-6AB499302301.activeSandbox/Root (1 items) to /
Aug 17 17:00:34 remtmay02 installd[428]: PackageKit: Executing script "./postinstall" in /private/tmp/PKInstallSandbox.1CxWbA/Scripts/com.getchef.pkg.chefdk.HIG1gN
Aug 17 17:00:34 remtmay02 installd[428]: ./postinstall: ln: /usr/bin/chef: Operation not permitted
Aug 17 17:00:34 remtmay02 installd[428]: ./postinstall: postinstall: Cannot link chef to /usr/bin
Aug 17 17:00:34 remtmay02 install_monitor[863]: Re-included: /Applications, /Library, /System, /bin, /private, /sbin, /usr
Aug 17 17:00:34 remtmay02 installd[428]: PackageKit: releasing backupd
Aug 17 17:00:34 remtmay02 installd[428]: PackageKit: allow user idle system sleep
Aug 17 17:00:34 remtmay02 installd[428]: PackageKit: Install Failed: Error Domain=PKInstallErrorDomain Code=112 "An error occurred while running scripts from the package “chefdk-0.7.0-1.pkg”." UserInfo={NSFilePath=./postinstall, NSURL=file://localhost/Volumes/Chef%20Development%20Kit/chefdk-0.7.0-1.pkg#chefdk-core.pkg, PKInstallPackageIdentifier=com.getchef.pkg.chefdk, NSLocalizedDescription=An error occurred while running scripts from the package “chefdk-0.7.0-1.pkg”.} {
	    NSFilePath = "./postinstall";
	    NSLocalizedDescription = "An error occurred while running scripts from the package \U201cchefdk-0.7.0-1.pkg\U201d.";
	    NSURL = "file://localhost/Volumes/Chef%20Development%20Kit/chefdk-0.7.0-1.pkg#chefdk-core.pkg";
	    PKInstallPackageIdentifier = "com.getchef.pkg.chefdk";
	}
Aug 17 17:00:34 remtmay02 installd[428]: PackageKit: Running idle tasks
Aug 17 17:00:34 remtmay02 Installer[858]: install:didFailWithError:Error Domain=PKInstallErrorDomain Code=112 "An error occurred while running scripts from the package “chefdk-0.7.0-1.pkg”." UserInfo={NSFilePath=./postinstall, NSURL=file://localhost/Volumes/Chef%20Development%20Kit/chefdk-0.7.0-1.pkg#chefdk-core.pkg, PKInstallPackageIdentifier=com.getchef.pkg.chefdk, NSLocalizedDescription=An error occurred while running scripts from the package “chefdk-0.7.0-1.pkg”.}
Aug 17 17:00:34 remtmay02 installd[428]: PackageKit: Removing client PKInstallDaemonClient pid=858, uid=503 (/System/Library/CoreServices/Installer.app/Contents/MacOS/Installer)
Aug 17 17:00:34 remtmay02 installd[428]: PackageKit: Done with sandbox removals
Aug 17 17:00:35 remtmay02 Installer[858]: Install failed: The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance.
Aug 17 17:00:35 remtmay02 Installer[858]: IFDInstallController 417553D0 state = 8
Aug 17 17:00:35 remtmay02 Installer[858]: Displaying 'Install Failed' UI.
Aug 17 17:00:35 remtmay02 Installer[858]: 'Install Failed' UI displayed message:'The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance.'.
```

This is because of the read-only `/usr`; we'll need to shift to dropping stuff into `/usr/local`

**Comment by mhenrixon** [comment]
:+1: on shifting stuff to `/usr/local` I think this is a problem with rootless

> System Integrity Protection, also called "rootless," is a new system security feature that prevents the user or any process from writing in system-protected folders. This list includes /System, /bin, /usr (but not /usr/local), and /sbin. Not even administrators can add to these folders or edit files that are in them, though they retain their access to the rest of the files on the drive. That's one way to protect important operating system files from external tampering!

**Comment by schisamo** [comment]
We should just replicate the fix @jaymzh and @natewalck provided for fixing this issue on Chef: https://github.com/chef/omnibus-chef/pull/439

**Comment by cassianoleal** [comment]
I've created a PR for that but it appears the system does not recognise my CCLA -- https://github.com/chef/omnibus-chef/pull/466

I can see my GitHub account authorised on Supermarket and the CCLA signed on June 23, 2014. Anything else I need to do?

**Comment by jeffbyrnes** [comment]
I believe this is fixed by #466.

**Comment by schisamo** [comment]
Yes this has been fixed.
